apps/frontend/src/libs/components/header/styles.module.css

```
@@ -0,0 +1,42 @@
+.header {
+	display: flex;
+	align-items: center;
+	justify-content: space-between;
+	padding: 10px 20px;
+	color: var(--color-text-primary);
+	background-color: var(--color-background-primary);
+}
+
+.logo_container {
```

we use kebab case for class names
```suggestion
.logo-container {
```